Casks/s/shadow-bot.rb

```diff
@@ -0,0 +1,35 @@
+cask "shadow-bot" do
+  arch arm: "-arm64", intel: ""
```

Comment:
```suggestion
  arch arm: "-arm64"
  livecheck_query = on_arch_conditional arm: "arm64", intel: "intel"
```

Accompanied by changes to `livecheck` below.